biz.aQute.launchpad/bnd.bnd

```diff
@@ -28,14 +28,16 @@ Import-Package: \
 -maven-scope: provided
 
 -buildpath: \
-    osgi.annotation;version=latest,\
-    osgi.core;version=latest,\
-    org.osgi.service.component;version='[1.4,1.5)',\
-    aQute.libg;version=latest, \
-    org.osgi.service.component.annotations;version='[1.3,1.4)',\
-    biz.aQute.bndlib;version=latest;packages=*,\
-    org.eclipse.jdt.annotation;version=@2, \
-    ${junit}
+	osgi.annotation;version=latest,\
```

Comment:
You did not change most of these lines yet they can whitespace changes. Perhaps you can remove the whitespace only changes here?

Reply:
Done